website/mails/mails.py

```diff
@@ -251,6 +251,8 @@ def get_english_article(word):
 REQUEST_EXPORT = Mail('support_request', subject='[via OSF] Export Request')
 REQUEST_DEACTIVATION = Mail('support_request', subject='[via OSF] Deactivation Request')
 
+REQUEST_DEACTIVATION_COMPLETE = Mail('request_deactivation_complete', subject='[via OSF] Deactivation Request')
```

Comment:
I'd recommend changing the subject to something like `OSF account deactivated` instead of using the same subject we send to the help desk?